examples/update-sshkey-secret

```diff
@@ -61,15 +55,14 @@ fi
 vault token-lookup > /dev/null 2>&1
 authenticate
 TMPDIR=$(mktemp -d -u)
-mkdir -p $TMPDIR
-cd $TMPDIR
+mkdir -p "$TMPDIR"
+
+if ssh-keygen -f "$TMPDIR"/private_ssh_key -N ''; then
+  base64 "${TMPDIR}/private_ssh_key" \
+  | vault write "${BASE_PATH}${PIPELINE}/private_ssh_key" value=-
 
-ssh-keygen -f ./private_ssh_key -N ''
-if [ $? -eq 0 ] ; then
-  echo -n $(cat private_ssh_key | base64) | vault write ${BASE_PATH}${PIPELINE}/private_ssh_key \
-    value=-
-  echo -n $(cat private_ssh_key.pub | base64) | vault write ${BASE_PATH}${PIPELINE}/private_ssh_key.pub \
-    value=-
+  base64 "${TMPDIR}/private_ssh_key.pub" \
```

Comment:
Same thing here, no need to base64 encode the secret since it's no longer required to process the secrets